.github/workflows/R-CMD-check.yaml

```diff
@@ -57,5 +64,3 @@ jobs:
           needs: check
 
       - uses: r-lib/actions/check-r-package@v2
-        with:
-          upload-snapshots: true
```

Comment:
We want these lines. I.e. this a new addition from upstream that we want to incorporate.